libs/coin-evm/src/signer.ts

```diff
@@ -25,7 +25,22 @@ type LedgerEthTransactionResolution = {
   domains: DomainServiceResolution[];
 };
 
+// Duplicate type definition from hw-app-eth.
```

Comment:
So that's a `FIXME` comment, out of scope for the migration but to address with the signer abstraction topic maybe.

Reply:
Yeah I followed what Stephane did above, I think a quick fix could be to move those types in the `types-live` lib if we want to avoid the dups 🤷‍♂️

Reply:
Not quite a Live type... I'd say for now `hw-app-eth` is the legitimate owner of that type?
So either importing the type from there, or a FIXME comment.

Reply:
Yes, the legitimate owner is `hw-app-eth` for both `LoadConfig` & `LedgerEthTransactionResolution` above.
I'm not sure creating a separate "types-XXX" for every lib is really the way to scale this, specifically when it's purely for DX so it's a dev dependency anyway.
A good old "@ledgerhq/types` would be fine to me, with proper folders to represent our libs and every "common" things to share can be there 🤷‍♂️ 
So yeah, in the end, I'm either it's a FIXME, either I add a new "types-hw-app-eth", or it's in `types-live`

Reply:
I meant just importing them from `hw-app-eth`, as it is now?
Neither `@ledgerhq/types` nor `types-hw-app-eth`, just keeping it simple but avoiding types duplication.

Reply:
The whole signer abstraction makes it kinda antipattern to import 'hw-app-eth' unfortunately :(